jsk_spot_robot/spoteus/spot-interface.l

```diff
@@ -318,6 +331,10 @@
           nil
         (send msg :data))
       ))
+  (:set-current-spot (node_id)
+    "set current node id"
```

Comment:
I don't  know why but sometimes eus returns the error when string type data was set after method name, so please use comment out or fix like https://github.com/sktometometo/jsk_robot/pull/19

Reply:
#19 is saying something like below...?
```
(:set-current-spot
  "documentation"
    (args) 
    ...)      ;; return error
```
```
(:set-current-spot (args)
   "documentation" ...) ;; ok
```
If you are talking about the same thing as in the example given above, then I think the current writing style should be fine...

Reply:
oh, I misunderstood, yeah, it's no problem. sorry...